fenix/app/src/main/java/org/mozilla/fenix/onboarding/OnboardingFragment.kt

```diff
@@ -72,8 +73,14 @@ class OnboardingFragment : Fragment() {
             activity?.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_PORTRAIT
         }
         val filter = IntentFilter(WidgetPinnedReceiver.ACTION)
-        LocalBroadcastManager.getInstance(requireContext())
+        LocalBroadcastManager.getInstance(context)
             .registerReceiver(pinAppWidgetReceiver, filter)
+
+        if (shouldShowDefaultBrowserCard(context) &&
```

Comment:
Now we can actually not show default browser card but only the dialog, we should rename this function to `isNotDefaultBrowser` or similar so this reads better.  Since this is used in two places now and implies that, previously it mapped to `shouldShowDefaultBrowserCard` behaviour.